src/AmbientSounds.Uwp/Strings/ru-RU/Resources.generated.cs

```diff
@@ -0,0 +1,906 @@
+// File generated automatically by ReswPlus. https://github.com/DotNetPlus/ReswPlus
```

Comment:
Did you add this file manually? I don't think we're supposed to have multiple copies of this file. We already have one so can you delete this?

Reply:
As. I've added it by manually because the localization didn't work by some reason. Now I pushed the commit to delete this one. Also I've added myself to the end of README.